tests/bluetooth/bsim_bt/bsim_test_eatt/src/common.c

```diff
@@ -0,0 +1,175 @@
+/* common.c - Common test code */
+
+/*
+ * Copyright (c) 2022 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "common.h"
+
+struct bt_conn *default_conn;
+
+static const struct bt_data ad[] = {
+	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+};
+
+static bool volatile is_connected;
+int volatile num_eatt_channels;
```

Comment:
Should be `volatile int`, not `int volatile` as per coding style conventions

Reply:
Didn't know that. Where are this mentioned? 

I guess this also applies to the `bool` just above?

It should probably also be `static`, right?

Reply:
I'm not sure if it's anywhere explicitly stated, but `volatile <type>` is the common convention, if you check for other places in the tree. It also reads much more naturally if you consider it from an English language perspective ("variable X is a volatile integer").

Reply:
As for `static` - I think I saw it exported as an extern in some header file

Reply:
- `num_eatt_channels` is checked in the main test procedure, so cannot be `static`.
- Can change to `volatile int`, as it matches west-const used elsewhere.